### PR TITLE
✨Source Shortio: Migrate Python CDK to Low-code CDK

### LINKED ISSUE #28119: short.io
http://short.io/

### PR DESCRIPTION
**Pull request by btkcodedev**
## What
*Migrating Source Shortio to Low-Code CDK*
Closes #28119

## How
*Developed using (Configuration Based Source) low-code CDK*

## Recommended reading order
1. `spec.yaml`
2. `manifest.yaml`
3. `schemas/*`

## Tests

<details open><summary><strong>Integration & Acceptance</strong></summary>
Full Test Results

image

</details>

## 🚨 User Impact 🚨
- No breaking changes, migration to low-code
<details><summary><strong>Migrate Connector: Source Shortio</strong></summary>

### Community member or Airbyter

- [x] **Community member?** Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [x] Secrets in the connector's spec are annotated with `airbyte_secret`
- [x] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [x] Code reviews completed
- [x] Documentation updated
    - [x] Connector's `README.md`
    - [x] Connector's `bootstrap.md`
    - [x] `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
    - [x] `docs/integrations/README.md`
    - [x] `airbyte-integrations/builds.md`
- [x] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

</details>

### CODE REVIEW

File: airbyte-integrations/connectors/source-shortio/source_shortio/spec.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+documentationUrl: https://developers.short.io/reference
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-shortio/source_shortio/spec.yaml:1]: Url to Airbyte connector page not the connector API refeenc.

File: airbyte-integrations/connectors/source-shortio/source_shortio/spec.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Shortio Spec
+  type: object
+  required:
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-shortio/source_shortio/spec.yaml:6]: additionalProperties: true

File: airbyte-integrations/connectors/source-shortio/source_shortio/manifest.yaml [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+      requester:
+        $ref: "#/definitions/v2_api_requester"
+
+  links_stream:
+    $ref: "#/definitions/v1_base_stream"
+    name: "links"
+    primary_key: "id"
+    $parameters:
+      extractor_path: "links"
+      path: "links"
```
Comment (marcosmarxm) [inline airbyte-integrations/connectors/source-shortio/source_shortio/manifest.yaml:58]: This stream is missing pagination.
image

Comment (marcosmarxm) [reply airbyte-integrations/connectors/source-shortio/source_shortio/manifest.yaml:58]: Also the `links` allow to use the parameter `afterDate`. The parameter `start_date`is not been used in this connector yet. You can implement the incremental sync method here.

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
Left some comments.

**Comment by marcosmarxm** [comment]
Also please resolve the conflicts.

**Comment by btkcodedev** [comment]
All comments resolved
Test - Passed state

image

**Review by marcosmarxm — APPROVED** [review]
Amazing work @btkcodedev

**Pull request merged**